pkg/reconciler/instances/istio/reset/label/action.go

```diff
@@ -0,0 +1,82 @@
+package label
+
+import (
+	"context"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/avast/retry-go"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/instances/istio/reset/data"
+	"github.com/pkg/errors"
+
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/instances/istio/consts"
+
+	"go.uber.org/zap"
+	"k8s.io/client-go/kubernetes"
+	k8sRetry "k8s.io/client-go/util/retry"
+
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/instances/istio/reset/pod"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+//go:generate mockery --name=Action --outpkg=mocks --case=underscore
+type Action interface {
+	Run(ctx context.Context, logger *zap.SugaredLogger, client kubernetes.Interface, retryOpts []retry.Option, sidecarInjectionByDefaultEnabled bool) error
+}
+
+// DefaultResetAction assigns pods to handlers and executes them
+type DefaultLabelAction struct {
+	gatherer data.Gatherer
+	matcher  pod.Matcher
+}
+
+func NewDefaultPodsLabelAction(gatherer data.Gatherer, matcher pod.Matcher) *DefaultLabelAction {
+	return &DefaultLabelAction{
+		gatherer: gatherer,
+		matcher:  matcher,
+	}
+}
+
+func labelWithWarning(context context.Context, kubeClient kubernetes.Interface, retryOpts wait.Backoff, podsList v1.PodList, log *zap.SugaredLogger) error {
+	for _, podToLabel := range podsList.Items {
+		if podToLabel.Namespace == consts.KubeSystem || podToLabel.Namespace == consts.KymaIntegration || podToLabel.Namespace == consts.KymaSystem {
```

Comment:
kube-system, kube-public and istio-system namespaces are already excluded by https://github.com/kyma-incubator/reconciler/blob/main/pkg/reconciler/instances/istio/reset/data/gatherer.go#L169
Shall we check them here again?

Reply:
No need to, I removed the check for kube-system